codecept.json

```diff
@@ -4,16 +4,12 @@
     "WebDriver": {
       "url": "{PMM_URL_HERE}",
       "browser": "chrome",
-      "windowSize": "2560x1600",
```

Comment:
@Kartikeya99 I need your changes on top of this, so can you please update it? I added these change to make the tests work with our server